docs/guides/self_hosting.md

```diff
@@ -200,4 +200,7 @@ $ curl -X POST 'http://localhost:5000/v1/llm/completions-sync?model_endpoint_nam
 You should get a response similar to:
 ```
 {"status":"SUCCESS","outputs":[{"text":". Tell me a joke about AI. Tell me a joke about AI. Tell me a joke about AI. Tell me","num_completion_tokens":30}],"traceback":null}
-```
\ No newline at end of file
+```
+
+### Pointing LLM Engine client to use self-hosted infrastructure
+The `llmengine` client makes requests to Scale AI's hosted infrastructure by default. You can have `llmengine` client make requests to your own self-hosted infrastructure by setting the `LLM_ENGINE_BASE_PATH` environment variable to the URL of the `llm-engine` pod. The exact URL of `llm-engine` pod depends on your Kubernetes cluster networking setup.
```

Comment:
> The exact URL of `llm-engine` pod depends on your Kubernetes cluster networking setup.

This might be true, but I think we have some default in our helm chart? @song-william @phil-scale 

Also, let's add a code snippet example, which assumes this default.

Reply:
@yixu34 yep you're right! The default cluster domain is specified here
https://github.com/scaleapi/llm-engine/blob/point-client-to-self-hosted/charts/llm-engine/values_sample.yaml#L99